index.js

```diff
@@ -1,13 +1,36 @@
 /***** Deliverable 1 *****/
+// const header = document.getElementById("header")
 const header = document.querySelector("#header")
-console.log("Here's your header:", header)
-
+header.innerHTML = "Welcome to Ian's Shiverpool PC Fan Club"
```

Comment:
`.innerHTML` works here, but if you're just putting a string inside an element (and not creating new DOM elements), `.textContent` would work as well:

```js
header.textContent = "Welcome to Ian's Shiverpool PC Fan Club"
```